### PR TITLE
Update form linking UI

### PR DESCRIPTION
**Pull request by proteusvacuum**
I was finding this UI really hard to use...

@millerdev 
## Before

screenshot from 2016-10-09 21-54-26
## After

screenshot from 2016-10-09 21-52-37

### CODE REVIEW

**Comment by benrudolph** [comment]
nice making good cleanups! i've always wondered why the select dropdown for advanced modules looks like it's from windows 95

File: corehq/apps/app_manager/templates/app_manager/partials/form_workflow.html [hunk 1]
```diff
@@ -54,10 +63,14 @@
...
-                <i class="fa fa-arrow-right"></i>
+        </div>
+        <div class="row">
+            <div class="col-sm-12 text-center">
+                <i class="fa fa-arrow-down"></i>
             </div>
-            <div class="col-sm-5">
+        </div>
+        <div class="row">
+            <div class="col-sm-11">
```
Comment (benrudolph) [inline corehq/apps/app_manager/templates/app_manager/partials/form_workflow.html]: how come this isn't 12?

**Comment by benrudolph** [comment]
👍 the only thing that's a bit odd is that the dropdown spacing doesn't match the xpath expression text box. definitely an improvement though so happy to merge

**Comment by proteusvacuum** [comment]
The dropdown has padding on it that made it overflow the panel. I gave up
trying to remove it without adding inline styles.

**Pull request merged**